skglm/estimators.py

```diff
@@ -952,6 +957,174 @@ def fit(self, X, y):
         return _glm_fit(X, y, self, Quadratic(), penalty, solver)
 
 
+class SCADRegression(LinearModel, RegressorMixin):
+
+    r"""Linear regression with SCAD penalty estimator.
+
+    The optimization objective for SCADRegression is, with :math:`x >= 0`:
+
+    .. math::
+        "pen"(x) = {
+            (alpha x                  , if \ \ \ \ \ \ \ \ \ \ x <= alpha),
+            (2 alpha gamma x - x^2 - alpha^2 / 2 (gamma - 1)
+                                      , if       alpha \ \   < x <= alpha gamma),
+            (alpha^2 (gamma + 1) / 2  , if       alpha gamma < x  )
+        :}
+
+    .. math::
+        "value" = sum_(j=1)^(n_"features") "pen"(abs(w_j))
+
+    For more details see
+    `Coordinate descent algorithms for nonconvex penalized regression,
+    with applications to biological feature selection, Breheny and Huang
+    <https://doi.org/10.1214/10-aoas388>`_.
+
+    Parameters
+    ----------
+    alpha : float, optional
+        Penalty strength.
+
+    gamma : float, default=3.7
+        If `` gammma = 1 `` this corresponds to hard thresholding
+
+        If `` gamma = np.inf `` his corresponds to soft thresholding
+
+
+    weights : array, shape (n_features,), optional (default=None)
+        Positive weights used in the L1 penalty part of the Lasso
```

Comment:
this si copied from somewhere else, but there is no L1 part here, neither is there a Lasso